NotifyUI/Info.plist

```diff
@@ -28,6 +28,7 @@
 			<array>
 				<string>auto_authorized_identifier</string>
 				<string>authorize_identifier</string>
+				<string>authorize_temporal_identifier</string>
```

Comment:
Does this need to go in any other plist as well?

Reply:
Nope